namerd/iface/interpreter-thrift/src/main/scala/io/buoyant/namerd/iface/ObserverCache.scala

```diff
@@ -32,11 +34,14 @@ class MaximumObservationsReached(maxObservations: Int)
  *                       instead.
  * @param inactiveCapacity The maximum size of the inactive cache.  LRU eviction is used to
  *                         maintain this constraint.
+ * @param inactiveTTLSecs The amount of time, in seconds, to keep observer in inactive cache before
+ *                        expiring them.
  * @param mkObserver The function to use to create new Observers if they are not in either cache.
  */
 class ObserverCache[K <: AnyRef, T](
   activeCapacity: Int,
   inactiveCapacity: Int,
+  inactiveTTLSecs: Long,
```

Comment:
I noticed here that the type for `inactiveTTLSecs` was `Long` but in the `ThriftIntepretererConfig`, the type for `inactiveTTLSecs` is of type `Option[Int]`. Should these two variables be of the same base type? As in `Long` for `inactiveTTLSSecs` in `ObserverCache` and `Option[Long]` for `inactiveTTLSecs` in `ThriftIntepreterInterfaceConfig`?

Reply:
You have a right. It is my mistake.